lesson3/src/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Мобильные телефоны</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header class="header">
+      <span>Мой магазин</span>
```

Comment:
```suggestion
      <a href="/">мой магазин</a>
```

в шапке название сайта это обычно ссылка на главную страницу